### PR TITLE
jira: fix silent epic error during unit tests

### PR DESCRIPTION
**Pull request by valentijnscholten**
during unit tests every push to JIRA resulted in an error because dojo was trying to add the issue to an epic, which didn't exist.
this error is swallowed silently by dojo so it was not failing the tests, but the logs were insinuating something was wrong.

most of the files changed are just re-recordings of the jira traffic

### CODE REVIEW

**Review by madchap — APPROVED** [review]

File: dojo/unittests/test_jira_import_and_pushing_api.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def assert_cassette_played(self):
             self.assertTrue(self.cassette.all_played)
 
     def _get_vcr(self, **kwargs):
-        my_vcr = super(JIRAConfigAndPushTestApi, self)._get_vcr(**kwargs)
-        my_vcr.record_mode = 'once'
+        my_vcr = super(JIRAImportAndPushTestApi, self)._get_vcr(**kwargs)
+        my_vcr.record_mode = 'none'
```
Comment (Maffooch) [inline dojo/unittests/test_jira_import_and_pushing_api.py:48]: Was this change intended?

Comment (valentijnscholten) [reply dojo/unittests/test_jira_import_and_pushing_api.py:48]: I think none is the right one for running existing unit tests

**Review by Maffooch — APPROVED** [review]
